Orange/widgets/model/owsgd.py

```diff
@@ -76,7 +76,7 @@ class Outputs(OWBaseLearner.Outputs):
     learning_rate_index = Setting(0)
     eta0 = Setting(.01)
     power_t = Setting(.25)
-    n_iter = Setting(5)
+    max_iter = Setting(10)
```

Comment:
Please implement a `migrate_settings` method. Without it, old workflows won't use the old `n_iter` value as the `max_iter` value while their semantics are essentially the same.

Reply:
I don't know how to do it.
Is this correct (added in class OWSGD)?

```
    @classmethod
    def migrate_settings(cls, settings_, version):
        if version < 2:
            settings_["max_iter"] = settings_.get("n_iter", 5)
```

Reply:
That should do the trick. Perhaps delete the setting `n_iter` as well to clean up? You can do this in one line `settings_["max_iter"] = settings_.pop("n_iter", 5)`.